### PR TITLE
[velero] Add labels to service account

### PR DESCRIPTION
**Pull request by hamza3202**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

**Review by jenting — COMMENTED** [review]
Thank u for your contribution to Velero, just one comment.

File: charts/velero/templates/serviceaccount-server.yaml [hunk 1]
```diff
@@ -12,4 +12,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ include "velero.chart" . }}
+{{- if .Values.serviceAccount.server.labels }}
+{{ toYaml .Values.serviceAccount.server.labels | nindent 4 }}
+{{- end }}
```
Comment (jenting) [inline charts/velero/templates/serviceaccount-server.yaml:17]: nit: I'd prefer use `with` other than `if` when there is no `else` condition.
```suggestion
{{- with .Values.serviceAccount.server.labels }}
  {{- toYaml . | nindent 4 }}
{{- end }}
```

Comment (hamza3202) [reply charts/velero/templates/serviceaccount-server.yaml]: Done

**Review by jenting — APPROVED** [review]
LGTM, thank u.

**Comment by hamza3202** [comment]
@carlisia @cpanato I need one more review on this PR.

**Review by carlisia — APPROVED** [review]

**Pull request merged**